### ISSUE TITLE
Focus styling on toolbar

### ISSUE THREAD

**Issue by cehfisher** [thememachine/eleven#162]
# The issue is:
Similar issue to #160, but on the very top level. On the toolbar (Manage, Shortcuts, etc) if you use your keyboard to tab, you will see no visible focus indicator. A quick code snippet shows that outline is set to none, which is something you want to remedy right away. This goes beyond "just" an accessibility issue, this should be considered a UX issue as well as it is difficult to tell where you are on the page.

Sample code in your project where outline is set to none:
```
#toolbar-bar nav a, #toolbar-bar .menu a, #toolbar-bar .toolbar-icon, #toolbar-bar .toolbar-menu a {
    outline: none;
}
```

# Where (url)
All admin level pages.

Todo
You can easily reset all focus indicator styles to use the default browser styles:
```
 {
   outline: auto 2px Highlight; // for non-webkit browsers
   outline: auto 5px -webkit-focus-ring-color; // for webkit browsers
 }
```
OR you can create a global focus style for all elements that matches your site theme a bit more. The key is to 1. have a visual focus indicator and 2. keep it as consistent as possible for all the different elements.

See this component for more details: http://a11y-style-guide.com/style-guide/section-general.html#kssref-general-link-focus

**Comment by mortendk** [comment]
toolbar focus reinstalled :)
